dc/council/code/titles/38/sections/38-2101.xml

```diff
@@ -3,11 +3,13 @@
   <num>38-2101</num>
   <heading>Authority to enter into agreement.</heading>
   <text>The Mayor of the District of Columbia is authorized to enter into and execute on behalf of the District of Columbia an agreement with any state or states legally joining therein in the form substantially as follows:</text>
+  <text>
```

Comment:
Labeling this just in case. This is one of those for which all the subelements might be more appropriately labeled as `<text>` as opposed to `<para>`.